docs/building/windows-full.md

```diff
@@ -70,7 +70,7 @@ What can you do with the produced binaries?
 
 **Important**: "We don’t support production deployments of these binaries on any platform". For PowerShell .NET (aka: FullCLR PowerShell) our recommendation is to continue using the PowerShell .NET version already shipping in Windows Client and Windows Server.
 
-The primary reason to build the PowerShell FullCLR binaries is to test backward compatibility, and interoperability between .NET and CoreCLR.  It is also important to mention some features like PowerShell Workflows are not currently available in the CoreCLR version. So we want to provide the ability for the Community to test CoreCLR PowerShell code changes while validating that these changes don't introduce regressions in .NET PowerShell (aka: as FullCLR PowerShell)
```

Comment:
@alexandair : if I made a PR that *only* split all .md files semantically, is that likely to be accepted quickly (i.e. w/o lots of complaints about unrelated content)? (I still have quite a few commits/PRs before I'm done w/ the spelling branch, so I'm not particularly eager, and I haven't even had a simple PR accepted)